lib/epicenter_web/live/add_visit_live.html.heex

```diff
@@ -0,0 +1,29 @@
+<div
+  data-case-investigation-tid={"#{@case_investigation_tid}"}
+  data-page="add-visit"
+  data-place-address-tid={"#{@place_address_tid}"}
+  id="add-visit-page"
+>
+  <section>
+    <header id="subnav">
+      <%= live_redirect to: Routes.place_search_path(EpicenterWeb.Endpoint, EpicenterWeb.PlaceSearchLive, @case_investigation), data: [role: "back-link"] do %>
```

Comment:
Let's add a todo to convert to `~p` verified routes in the future.

Reply:
added [update phoenix syntax using ~p · Issue #13 · RatioPBC/epi-viewpoint](https://github.com/RatioPBC/epi-viewpoint/issues/13)